### ISSUE TITLE
Documentation links not working

### ISSUE THREAD

**Issue by vcarballo** [OAI/OpenAPI-Specification#3197]
None of the local links in the https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md file work.

**Comment by MikeRalphson** [comment]
Thanks, we're aware and we were discussing it on today's call. We have agreed what we need to do to fix.

**Comment by gevial** [comment]
Same for version 3.0.1.

**Comment by MikeRalphson** [comment]
It affects all the markdown files, same cause.

**Comment by handrews** [comment]
We discussed this in the [March 9th meeting](https://github.com/OAI/OpenAPI-Specification/issues/3183#issuecomment-1456609127) and decided to fix these things in place.  @webron was going to figure out the exact logistics, which has not happened by the time I commented in the agenda for the [April 27th meeting](https://github.com/OAI/OpenAPI-Specification/issues/3253#issuecomment-1523800012).  AFAIK, this last detail was not resolved.

One difficulty was the range of PRs from various people that needed tweaks, but the PR submitters had moved on due to lack of activity.

Since it's been a good 6 months without movement, I'm going to consolidate the PRs myself.
